<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paradialog_rigidaffine</class>
 <widget class="QDialog" name="Paradialog_rigidaffine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_fileio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>File IO: </string>
   </property>
   <widget class="QLabel" name="label_img_tar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input target/template image: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input subject image: (to be warpped)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_img_tar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_img_tar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>46</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_img_sub">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>97</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_img_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_img_sub2tar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_sub2tar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output warpped subject image: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_img_sub2tar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>146</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_swc_grid">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>197</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_swc_grid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output warpped grid to swc: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_swc_grid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_paras">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>451</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Paras:</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_refchannel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>36</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_refchannel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference channel:
(R:1, G:2, B:3)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_downsampleratio">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_downsample">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Downsample ratio:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Gaussian">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>211</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Gaussian smoothing</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_Gau_sigma">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>52</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>kernel sigma: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Gau_radius">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>kernel radius:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Gau_radius">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>25</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Gau_sigma">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_histogrammatching">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram matching</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_alignedges">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Align edges</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_iter_max">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_iter_max">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max iter num:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_step_multiplyfactor">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Step inimultiplyfactor:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_step_multiplyfactor">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_step_annealingrate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Step annealing rate:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_step_annealingrate">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_step_min">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Min step (stop iter):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_step_min">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Paradialog_rigidaffine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Paradialog_rigidaffine</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
